docs/options.md

```diff
@@ -2,20 +2,21 @@
+| k8s-tag-cluster-id        | aws-cluster-id-1        |                                                  | ID of the Kubernetes cluster used for tagging provisioned EBS volumes|
+| aws-sdk-debug-log         | true                    | false                                            | If set to true, the driver will enable the aws sdk debug log level|
+| logging-format            | json                    | text                                             | Sets the log format. Permitted formats: text, json|
+| user-agent-extra          | csi-ebs                 | helm                                             | Extra string appended to user agent|
+| enable-otel-tracing       | true                    | false                                            | If set to true, the driver will enable opentelemetry tracing. Might need [additional env variables](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#general-sdk-configuration) to export the traces to the right collector|
+| batching                  | true                    | true                                             | If set to true, the driver will enable batching of API calls. This is especially helpful for improving performance in workloads that are sensitive to EC2 rate limits at the cost of a small increase to worst-case latency|
+| modify-volume-request-handler-timeout | 10s                     | 2s                                               | Timeout for the window in which volume modification calls must be received in order for them to coalesce into a single volume modification call to AWS. If changing this, be aware that the ebs-csi-controller's csi-resizer and volumemodifier containers both have timeouts on the calls they make, if this value exceeds those timeouts it will cause them to always fail and fall into a retry loop, so adjust those values accordingly.
+| warn-on-invalid-tag       | true                    | false                                            | To warn on invalid tags, instead of returning an error|
+| reserved-volume-attachments | 2                       | -1                                               | Number of volume attachments reserved for system use. Not used when --volume-attach-limit is specified. When -1, the amount of reserved attachments is loaded from instance metadata that captured state at node boot and may include not only system disks but also CSI volumes.|
+| legacy-xfs                | true                    | false                                            | Warning: This option will be removed in a future release. It is a temporary workaround for users unable to immediately migrate off of older kernel versions. Formats XFS volumes with `bigtime=0,inobtcount=0,reflink=0`, so that they can be mounted onto nodes with linux kernel ≤ v5.4. Note that XFS volumes formatted with this option will only have timestamp records until 2038.|
```

Comment:
Not sure if I maybe misunderstood this but, the last line you talk about `Note that XFS volumes formatted with this option will only have timestamp records until 2038` won't this brick the volume after 2038 ? And if so should we mention the implications of it here ?